### DISCUSSION TITLE
Add ID List to limit connection to self hosted Rustdesk Server

### DISCUSSION THREAD

**Discussion by jsantamaria197024** [rustdesk/rustdesk#7353] [General]
For limit the use of the Server Relay to a list of IDs, this for give more secure to Server Relay

Realy hope this feature

**Comment by xlionjuan** [comment]
If you add `-k _` arg to your hbbs and hbbr, it will only accept the connection with valid key.

**Comment by xlionjuan** [comment]
Someone suggested the server can support multiple keys before but I forgot where.
